verifiers/interfaceValidator.js

```diff
@@ -0,0 +1,85 @@
+"use strict";
```

Comment:
discussed offline, created a deliverable

---
In reply to: [87683181](https://github.com/openT2T/translators/pull/119#discussion_r87683181) [](ancestors = 87683181,87661373)